### ISSUE TITLE
Optional Chaining does not propagate receiver when parenthesized

### ISSUE THREAD

**Issue by jridgewell** [microsoft/TypeScript#35476]
**TypeScript Version:**  typescript@3.8.0-dev.20191203

**Search Terms:**
optional chaining receiver call parentheses

**Code**

```ts
const foo = { bar() { return this } };

console.assert(foo === (foo?.bar)());
```

**Expected behavior:**

`foo` should be propagated as the receiver to the `foo.bar` method call.

**Actual behavior:**

No receiver is propagated (due to the ternary breaking the receiver).

**Playground Link:** https://www.typescriptlang.org/play/?ssl=1&ssc=1&pln=4&pc=1#code/MYewdgzgLgBAZiEMC8MDeMBGBDATgCgEp0ZcBTKAV1zBigAsBLCGAXzYG4AoL0SEADZkAdNggQyuKPgRJk8mDMQB+YTlyEihDkA

**Related Issues:**

**Comment by ajafff** [comment]
Fix is up at #35494

**Comment by RyanCavanaugh** [comment]
@rbuckton please take a look at the linked PR. Thanks!
